test/integration/vaults-happy-path.integration.ts

```diff
@@ -139,10 +139,10 @@ describe("Scenario: Staking Vaults Happy Path", () => {
   });
 
   it("Should have vaults factory deployed and adopted by DAO", async () => {
-    const { stakingVaultFactory } = ctx.contracts;
+    const { stakingVaultFactory, stakingVaultBeacon } = ctx.contracts;
 
-    const implAddress = await stakingVaultFactory.implementation();
-    const adminContractImplAddress = await stakingVaultFactory.delegationImpl();
+    const implAddress = await stakingVaultBeacon.implementation();
+    const adminContractImplAddress = await stakingVaultFactory.DELEGATION_IMPL();
```

Comment:
I don't like naming here; it sounds confusing a bit; maybe it would be better to change to `const vaultAddress = ...` / `const delegationAddress = ...`? What do you think?

Reply:
thks! left over after refactoring